src/Core/Components/Avatar/FluentAvatar.razor.cs

```diff
@@ -0,0 +1,143 @@
+    public string? Image { get; set; }
+
+    /// <summary>
+    /// Provide custom initials rather than one generated via the name
+    /// <para>It is usually not necessary to specify custom initials; by default they will be derived from the name prop, using the getInitials function.</para>
+    /// </summary>
+    [Parameter]
+    public string? Initials { get; set; }
+
+    /// <summary>
+    /// The name of the person or entity represented by this Avatar. This should always be provided if it is available.
+    /// The name is used to determine the initials displayed when there is no image.It is also provided to accessibility tools.
+    /// </summary>
+    [Parameter]
+    public string? Name { get; set; }
+
+    /// <summary>
+    /// Indicates the shape of the avatar.
+    /// </summary>
+    [Parameter]
+    public AvatarShape? Shape { get; set; }
+
+    /// <summary>
+    /// Size of the avatar in pixels.
+    /// <para>Size is restricted to a limited set of supported values recommended for most uses (see AvatarSizeValue) and based on design guidelines for the Avatar control.
+    /// Note: At size 16, if initials are displayed, only the first initial will be rendered.
+    /// If a non-supported size is needed, set size to the next-smaller supported size, and set width and height to override the rendered size.
+    /// For example, to set the avatar to 45px in size: style={{ width: '45px', height: '45px' }}</para>
+    /// </summary>
+    [Parameter]
+    public AvatarSize? Size { get; set; }
+
+    /// <summary>
+    /// Command executed when the user clicks on the button.
+    /// </summary>
+    [Parameter]
+    public EventCallback<MouseEventArgs> OnClick { get; set; }
+
+    /// <summary>
+    /// Occurs when a key is pressed down while the component is focused.
+    /// </summary>
+    [Parameter]
+    public EventCallback<KeyboardEventArgs> OnKeyDown { get; set; }
+
+    /// <summary />
+    protected async Task OnClickHandlerAsync(MouseEventArgs e)
+    {
+        if (OnClick.HasDelegate)
+        {
+            await OnClick.InvokeAsync(e);
+        }
+
+        await Task.CompletedTask;
+    }
+
+    /// <summary />
+    protected async Task OnKeyDownHandlerAsync(KeyboardEventArgs e)
+    {
+        if (OnKeyDown.HasDelegate)
+        {
+            await OnKeyDown.InvokeAsync(e);
+        }
+
+        await Task.CompletedTask;
+    }
+
+    private string? GetInitialsValue => string.IsNullOrEmpty(Initials)
+        ? null
+        : Initials;
+
+    private string? GetNameValue => string.IsNullOrEmpty(Name)
+        ? null
+        : Name;
+
+    private bool DisplayIcon =>
+        Icon is not null
+        && string.IsNullOrEmpty(Name)
+        && string.IsNullOrEmpty(Initials);
+
+    private bool DisplayImage => !string.IsNullOrEmpty(Image);
+
+    private string GetAvatarSize =>
+        Size is not null
+        ? $"{(int)Size}px"
+        : "32px"; // Default component size
+
+    private string? GetActiveValue
```

Comment:
You can simplify to `private string? GetActiveValue => Active == true ? "active" : "inactive"`

Reply:
No we can't, the item has 3 differents styles : 
- Not defined 
- Active 
- Inactive

![image](https://github.com/user-attachments/assets/e80f7aa6-bf54-4fe7-a9cf-7535bf664848)

Please find my observation here : https://typescript-xbd5mxyq.stackblitz.io

Reply:
Weird :-) Maybe a `bool?` ? :-)